operators/salesforce_to_s3_operator.py

```diff
@@ -122,44 +125,47 @@ def __init__(self,
                  s3_conn_id,
                  s3_bucket,
                  s3_key,
-                 sf_fields=None,
                  fmt="csv",
                  query=None,
+                 query_fields=None,
+                 query_scope=None,
                  relationship_object=None,
                  record_time_added=False,
                  coerce_to_timestamp=False,
+                 stream=False,
                  *args,
                  **kwargs):
 
         super(SalesforceToS3Operator, self).__init__(*args, **kwargs)
 
         self.sf_conn_id = sf_conn_id
         self.object = sf_obj
-        self.fields = sf_fields
         self.s3_conn_id = s3_conn_id
         self.s3_bucket = s3_bucket
         self.s3_key = s3_key
         self.fmt = fmt.lower()
         self.query = query
+        self.query_fields = query_fields
+        self.query_scope = query_scope
         self.relationship_object = relationship_object
         self.record_time_added = record_time_added
         self.coerce_to_timestamp = coerce_to_timestamp
 
-    def special_query(self, query, sf_hook, relationship_object=None):
-        if not query:
-            raise ValueError("Query is None.  Cannot query nothing")
+    # def special_query(self, query, sf_hook, relationship_object=None):
```

Comment:
@benjamingregory Looks like you initially committed this, If it is in use with any customers let's have @codeinvain leave it in, otherwise let's remove the commented code.

Reply:
@andscoop I believe that was you -- I only committed it when I brought it over to this repo.